modules/aws/dra-admin/variables.tf

```diff
@@ -27,8 +27,8 @@ variable "key_pair" {
 
 variable "dra_version" {
   type        = string
-  default     = "4.13"
-  description = "The DRA version to install. Supported versions are 4.11.0.10.0.7 and up. Both long and short version formats are supported, for example, 4.11.0.10.0.7 or 4.11. The short format maps to the latest patch."
+  default     = "4.14"
+  description = "The DRA version to install. Supported versions are 4.11.0.10 and up. Both long and short version formats are supported, for example, 4.11.0.10 or 4.11. The short format maps to the latest patch."
```

Comment:
There's a bug in the description, the modules accept only full version format, it's neither long nor short. So it should be:
"The DRA version to install. Supported versions are 4.11.0.10.0.7 and up. Full version format is supported, for example, 4.11.0.10.0.7."

Reply:
"the modules accept only full version format" -> just to make sure, the default value "4.14" that appears one line before the "description", will still work?

Reply:
Yes, because the flow is:
1. User inputs 4.14 in example variable
2. In example code, when DRA module is instantiated, the example doesn't pass the version variable it received as it, it resolves it to the full version using the core/globals module.
3. The module accepts the full version